packages/sirius-web/frontend/sirius-web-application/src/diagrams/useDiagramFilterSubscription.tsx

```diff
@@ -65,7 +65,7 @@ export const useDiagramFilterSubscription = (
       const { diagramFilterEvent: payload } = gqlDiagramFilterEventSubscription;
       if (isFormRefreshedEventPayload(payload)) {
         const { form } = payload;
-        setState((prevState) => ({ ...prevState, form }));
+        setState((prevState) => ({ ...prevState, form, complete: false }));
```

Comment:
The `...prevState` should be enough isn't it? Same comment for all the other hooks.

Reply:
That was my first attempt, but without this explicit `complete: false` the Details view is always empty. It seems we receive an `onComplete` invocation at some point and then if we don't reset `complete` to false it stays `true`.

Reply:
The 'onComplete' call happens when I select a Document in the explorer.
When interactive with the explorer, the only way to find a proper element to select involves selecting a document as the first step, which does not produce a details view, hence the onComplete.

If I navigate to a diagram's page directly, and then select an element from the diagram, the Details view correclyt appears (without the explicit `complete: false`), but as soon as I re-select a Document, I'm back to a 'complete: true' state and without an explicit reset the view stays empty.

Reply:
Without the explicit `complete: false`:

https://github.com/user-attachments/assets/eb60b68a-107d-42e6-b15f-d088d1d1bf85